readme.md

```diff
@@ -1,2 +1,3 @@
 #S47_RANDOM
-This is a simple website that generates a random number on reload and will show a winning or lossing message
+This is a simple website that generates a random number on reload and will show a winning or lossing message.user don't need to do anything 
```

Comment:
It would be better to fix the typo in the readme file and improve the sentence structure for better readability. [medium]